code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -40,7 +40,12 @@
 /obj/item/clothing/mask/facehugger/attackby(obj/item/O, mob/user, params)
 	return O.attack_obj(src, user, params)
 
-/obj/item/clothing/mask/facehugger/attack_alien(mob/user) //can be picked up by aliens
+/obj/item/clothing/mask/facehugger/attack_alien(mob/user) //can be picked up by aliens with the variable can_grab_facehuggers set to true
+	if(istype(user, /mob/living/carbon/alien/humanoid))
+		var/mob/living/carbon/alien/humanoid/alium = user
+		if(!alium.can_grab_facehuggers)
+			to_chat(user, "<span class='warning'>Your claws are too sharp to handle your unborn sisters!</span>")
```

Comment:
What if I want to RP as a handsome boy alien ;c

Reply:
While making this I thought the same thing, but unfortunately canonically all Xenomorphs are female. You'll have to roll for trans RP if you want to become a handsome boy alien, maybe get some surgery.... can Xenomorphs get plastic surgery, anyway?